apps/docs/src/guide/contracts/deploying-contracts.md

```diff
@@ -7,40 +7,44 @@
 
 # Deploying Contracts
 
-This guide walks you through deploying a contract using the SDK, covering loading contract artifacts, initializing a contract factory, and deploying the contract.
+To deploy a contract using the SDK, you can use the `ContractFactory`. This process involves collecting the contract artifacts, initializing the contract factory, and deploying the contract.
 
-> **Note:** The maximum contract deployment size supported by the SDK is 100 KB. The SDK will `throw` an [error](../errors/index.md) for contracts larger than this size.
+The SDK utilizes two different deployment processes. Either simply using a single create transaction to deploy the entire contract bytecode, or, by splitting the contract into multiple chunks, deploying them as blobs (on chain data accessible to the VM) and then generating a contract from the associated blob IDs. That generated contract is then deployed as a create transaction.
 
-## 1. Obtaining Contract Artifacts
+The `ContractFactory` offers the following methods for the different processes:
 
-After writing a contract in Sway and compiling it with `forc build` (<a :href="indexUrl" target="_blank" rel="noreferrer">read more</a> on how to work with Sway), you will obtain two important artifacts: the compiled binary file and the JSON ABI file. These files are required for deploying a contract using the SDK.
+- `deploy` for deploying contacts of any size (will automatically choose the appropriate deployment process).
+- `deployContract` for deploying the entire contract bytecode in a single create transaction.
+- `deployContractAsBlobs` for deploying the contract in chunks as blobs, and then deploying the contract as a create transaction.
 
-## 2. Setting up the SDK Environment
+> **Note:** Due to the nature of blob deployments, both `deploy` and `deployContractAsBlobs` may require multiple transactions to deploy the contract.
 
-Before deploying a contract, set up the necessary environment by importing the required SDK components and initializing a wallet and a provider.
+The deployment process used by the `ContractFactory` is dependent on the size of the contract you are attempting to deploy. The threshold for the contract size is dictated by the chain. You can find the maximum contract size supported by the chain by querying the chains consensus parameters.
 
-<<< @/../../docs-snippets/src/guide/contracts/deploying-contracts.test.ts#contract-setup-1{ts:line-numbers}
+<<< @/../../docs-snippets/src/guide/contracts/deploying-contracts.test.ts#get-contract-max-size{ts:line-numbers}
 
```

Comment:
```suggestion
The SDK utilizes two different deployment processes, depending on the contract's size. The threshold for the contract size is dictated by the chain size and can be queried like this:

<<< @/../../docs-snippets/src/guide/contracts/deploying-contracts.test.ts#get-contract-max-size{ts:line-numbers}

It either uses a single create transaction to deploy the entire contract bytecode, or it splits the contract bytecode into multiple chunks, deploys them as blobs (on chain data accessible to the VM), and then generates a contract from the associated blob IDs. That generated contract is then deployed as a create transaction.

The `ContractFactory` offers the following methods for the different processes:

- `deploy` for deploying contacts of any size (will automatically choose the appropriate deployment process).
- `deployContract` for deploying the entire contract bytecode in a single create transaction.
- `deployContractAsBlobs` for deploying the contract in chunks as blobs, and then deploying the contract as a create transaction.

> **Note:** Due to the nature of blob deployments, both `deploy` and `deployContractAsBlobs` may require multiple transactions to deploy the contract.
```

Reply:
Note that this is just a take on the docs that flows better for me. Pick and choose what you will from it.

Reply:
Sorry @nedsalk I haven't missed this. Will compare today.

Reply:
It's nice 👍🏻